src/Bitfinex.Client.Websocket/Responses/Orders/OrderConverter.cs

```diff
@@ -74,10 +74,11 @@ public static OrderStatus ParseStatus(string status)
                 case var s when s.StartsWith("executed"):
                     return OrderStatus.Executed;
                 case "partially filled":
-                case var s when s.StartsWith("partially filled"):
+                case var s when s.Contains("partially filled"):
                     return OrderStatus.PartiallyFilled;
                 case "canceled":
-                case var s when s.StartsWith("canceled"):
+                case var s when s.Contains("canceled"):
+                case "insufficient balance":
```

Comment:
Thanks for PR. 
Shouldn't it be `var s when s.Contains("insufficient balance")`? 
Because there is `... 0,null,0,"INSUFFICIENT BALANCE (U1)",null, ...` in your log.

Reply:
insufficient balance can be no order made (canceled) and also partially filled as in log

Reply:
it can be either canceled or partially filled

Reply:
I know, but there is `case var s when s.Contains("partially filled"):` before so it will be recognized correctly. 

But here is some weird string ` (U1)` which ruin the switch `case "insufficient balance":` and it will end up as `OrderStatus.Undefined`

Reply:
I see what you mean.

This maybe then? 

`case var b when b.StartsWith("insufficient balance") && !b.Contains("partially filled"):`

Reply:
`&& !b.Contains("partially filled")` not needed, because it is already filtered above, see: 
![image](https://user-images.githubusercontent.com/3494837/50728731-18e05180-112f-11e9-89cc-2c8c87e2d730.png)

Reply:
And I would use `Contains` instead of `StartsWith`, it will be more secure